roles/jboss_fuse_on_karaf/tasks/main.yml

```diff
@@ -96,3 +96,66 @@
   become: true
   when: 'jboss_fuse'
 
+- name: Use locate to look for camel-ver
+  raw: locate camel-core | sed -n 's/.*\(redhat-[0-9]\{6\}\).*/\1/p'
+  register: jboss_fuse_camel_ver
+  ignore_errors: yes
+  when: 'have_locate and jboss_fuse'
+
+- name: check /system/org/apache/camel/camel-core for camel-ver
+  raw: ls -1 "{{ item }}"/system/org/apache/camel/camel-core 2>/dev/null | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
```

Comment:
Based on @noahl bash "globbing" identified issue (#1089)
I think this should be:

`ls -1 '{{ item }}/system/org/apache/camel/camel-core' 2>/dev/null`